fluent-plugin-enhance-k8s-metadata/lib/fluent/plugin/filter_enhance_k8s_metadata.rb

```diff
@@ -113,7 +115,33 @@ def normalize_param
         log.debug "bearer_token_file: '#{@bearer_token_file}', exist: #{File.exist?(@bearer_token_file)}"
 
         @cache_ttl = :none if @cache_ttl <= 0
-        log.info "cache_ttl: #{cache_ttl}, cache_size: #{@cache_size}"
+        log.info "cache_ttl: #{@cache_ttl}, cache_size: #{@cache_size}, cache_refresh: #{@cache_refresh}"
+      end
+
+      def start_cache_timer
+        timer_execute(:"cache_refresher", @cache_refresh) {
+          entries = @cache.to_a
+          log.info "Refreshing metadata for #{entries.count} entries"
+
+          entries.each { |entry|
+            begin
+              log.info "Refreshing metadata for key #{entry[0]}"
+              split = entry[0].split("::")
```

Comment:
should we wrap the body of the `entries.each` with a begin/rescue to avoid one bad entry preventing the refresh of subsequent entries?

Reply:
Never mind, you've already done this, Nice